lib/sqlalchemy/orm/events.py

```diff
@@ -1396,12 +1481,16 @@ def my_before_commit(session):
 
     _dispatch_target = Session
 
-    def _lifecycle_event(fn):
+    def _lifecycle_event(  # type: ignore [misc]
```

Comment:
IMHO we can add @staticmethod and delete "# type: ignore [misc]".
Also, if "-> Callable[[SessionEvents, Session, Any], None]" we can delete "# type: ignore [return-value]"